README.md

```diff
@@ -67,13 +67,13 @@ NeosModLoader is licensed under the GNU Lesser General Public License (LGPL). Se
 
 Third-party libraries distributed alongside NeosModLoader:
 
-- [LibHarmony] ([MIT License](https://github.com/pardeike/Harmony/blob/79d62b42c71d005b3cb1d94e741bfd4ce7e03a6b/LICENSE))
+- [LibHarmony] ([MIT License](https://github.com/pardeike/Harmony/blob/master/LICENSE))
```

Comment:
I had intentionally linked to a specific license version, so that in the event they relicense my link is still correct. Do you think that's worth doing?

Reply:
I mean, I don't imagine it'll be changed. Pinning it a version is fine, but a blob url to some very specific commit seems unintentional.